examples/28-wireframe/vs_wf_mesh_bc.sc

```diff
@@ -0,0 +1,20 @@
+$input a_position, a_normal
+$output v_view, v_normal
+
+/*
+ * Copyright 2016 Dario Manesku. All rights reserved.
+ * License: https://github.com/bkaradzic/bgfx/blob/master/LICENSE
+ */
+
+#include "../common/common.sh"
+#include "uniforms.sh"
+
+void main()
```

Comment:
If there is no difference in shaders, you should use existing ones instead to avoid duplication.